modules/ROOT/pages/log-search-query-syntax.adoc

```diff
@@ -89,10 +90,10 @@ To escape these characters, use the `\` before the character. For example, to se
 
 == Search Fields
 
-When performing a search you can either specify a field name or use the default field. The field name and default field are specific to the search implementation.
+When performing a search, either specify a field name or use the default field. The field name and default field are specific to the search implementation.
```

Comment:
"or use the default."
I don't think we need "field" after "default" in either of these cases, if by that we mean the default name rather than the field itself.
"provided default"? "default provided"?